.github/workflows/CI.yml

```diff
@@ -229,6 +229,8 @@ jobs:
       - name: "Coverage comment 💬"
         uses: py-cov-action/python-coverage-comment-action@v3
         id: coverage_comment
+        # avoid running this on schedule or releases
+        if: github.event.workflow_run.event != 'schedule' || github.event.workflow_run.event != 'release'
```

Comment:
```suggestion
        if: github.event.workflow_run.event != 'schedule' && github.event.workflow_run.event != 'release'
```

Reply:
I do not think the suggestion is correct, we want to skip the tests if the trigger was a `release` or a `schedule` trigger event.
This makes it so that it has to be both a `release` and `schedule` trigger which based on our workflows does not happen (we do not make automated releases based on a schedule)

Reply:
on the contrary I think the suggestion is correct.

> This makes it so that it has to be both a `release` and `schedule` trigger

No, it makes it so that it has to be *neither* a `release` *nor* a `schedule` trigger (because of the `!=` in each clause). Which is what we want right?

If the binary operator is `||` then *any* value for `event` will pass the conditional and trigger the workflow, because *any* value for `event` will be either not "schedule" or not "release".

Reply:
My bad I got confused with the != Operator. 
Thanks both for the patience.